common/views/components/EventPromo/EventPromo.js

```diff
@@ -13,7 +13,7 @@ import { CardOuter, CardBody } from '../Card/Card';
 import Divider from '../Divider/Divider';
 import styled from 'styled-components';
 
-const WatchWrapper = styled.div`
+export const WatchWrapper = styled.div`
```

Comment:
If we're using these elsewhere would it make sense to pull them into their own components for ease of navigation?